### DISCUSSION TITLE
Three recommendations

### DISCUSSION THREAD

**Discussion by akivec** [ftde0/yt2009#183] [General]
1. Can you add a better sign in system in the mobile app for someone to be able to sign in with a custom username with their session token and for commenting, liking, disliking, adding to playlists etc. to work on the app. I need this because i plan on using the app as my daily driver. 
2. Add new homepage option in mobile flags. There is already new related but new homepage will be easier for people to use as a daily driver. 
3. Make the home page show videos from your subscribed channels. 

I want these 3 things for the app to be easier to use as a daily driver. I hope these recommendations get added

**Comment by mrkrabs206** [comment]
https://github.com/ftde0/yt2009/commit/1b8c39a0c0744f4544e751d4866c50b8828c508b
